benchmarks/decoders/benchmark_decoders.py

```diff
@@ -13,7 +13,7 @@
 
 from benchmark_decoders_library import (
     AbstractDecoder,
-    BatchParameters,
+    # BatchParameters,
```

Comment:
Ignore changes here; once we fix these failures on trunk, I'll pull those changes. These changes just let this run without error.